Content.Server/Botany/Systems/MutationSystem.cs

```diff
@@ -373,7 +375,5 @@ private void CrossBool(ref bool val, bool other)
     }
 
     private bool Random(float p)
-    {
-        return _robustRandom.Prob(p);
-    }
+        => _robustRandom.Prob(p);
```

Comment:
It does not appear to be convention to rewrite single-line functions in lambda form, see: https://github.com/space-wizards/RobustToolbox/blob/master/Robust.Shared/Random/RobustRandom.cs

Reply:
(To be clear, in the rest of the codebase, methods that behave like getters and setters seem to be treated differently. But `Random(p)` is neither a getter nor setter.)

Reply:
OK, I will revert it.

Reply:
Done.